projects/examples/src/components/action-menu/action-menu.example.component.ts

```diff
@@ -192,4 +206,14 @@ export class ActionMenuExampleComponent<R extends Record, T extends HandlerData>
     toggleDropdownDisable(): void {
         this.isDropdownDisabled = !this.isDropdownDisabled;
     }
+
+    ngOnInit(): void {
+        mousetrap.bind(this.kbdShortcut, () => {
+            this.spotlightOpen = true;
+            // Since this happens outside angular zone we need to notify angular manually
+            // otherwise the user should wait for any operation that will trigger angular change detection (angular zone event)
+            this.changeDetectorRef.detectChanges();
```

Comment:
If you take a look in the core vcd-ui code there is a way to avoid this change detection. Basically by createin a new instance of the mousetrap in the constructor